go/ngql/types.go

```diff
@@ -439,9 +439,9 @@ func getListElements(v types.Value, args map[string]interface{}) interface{} {
 	var iter listOrSetIterator
 	switch l := l.(type) {
 	case types.List:
-		iter = l.IteratorAt(uint64(idx))
+		iter = l.IteratorAt(uint64(idx), false)
 	case types.Set:
-		iter = l.IteratorAt(uint64(idx))
+		iter = l.IteratorAt(uint64(idx), false)
```

Comment:
kinda icky, but maybe this should be:
```go
const readAheadThreshhold 100 // or whatever
iter = l.IteratorAt(uint64(idx), count >= readaheadThreshhold)
```
same elsewhere in ngql. It seems like a bummer to lose readahead entirely here.